mama/c_cpp/src/c/mama/msg.h

```diff
@@ -62,7 +64,8 @@ typedef enum mamaPayloadType_
     MAMA_PAYLOAD_EXEGY      = 'X',
 
 /* Unknown Payload */
-    MAMA_PAYLOAD_UNKNOWN    = 'U'
+    MAMA_PAYLOAD_UNKNOWN    = 'U',
+    MAMA_PAYLOAD_NULL       = '\0'
```

Comment:
It's used in mama.c, for example:

    693 mamaPayloadBridge
    694 mamaInternal_findPayload (char id)
    695 {
    696     mamaPayloadLib* payloadLib = NULL;
    697
    698     if (MAMA_PAYLOAD_NULL == id)
    699         return NULL;

So we are assuming NULL is undefined, and we can easily use the enum for checking values. I don't think `MAMA_PAYLOAD_ID_NULL` is any easier for people to use over the enum either, though that may just be me.